### DISCUSSION TITLE
Error when running docker using arm64: cron: unrecognized service

### DISCUSSION THREAD

**Discussion by ethanwanggg** [LycheeOrg/Lychee#2560] [Bugs]
### Lychee version

v5.5.1

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

-------------------------------------
  _               _                
 | |   _   _  ___| |__   ___  ___  
 | |  | | | |/ __|  _ \ / _ \/ _ \ 
 | |__| |_| | (__| | | |  __/  __/ 
 |_____\__, |\___|_| |_|\___|\___| 
       |___/                       

-------------------------------------
Lychee Version: 5.5.1 (nightly)
Lychee Branch:  master
Lychee Commit:  a50c04e
https://github.com/LycheeOrg/Lychee/commit/a50c04ea624f939e1c064ac2cfcf760fb5b6495d
-------------------------------------
**** Make sure the /conf /uploads /sym /logs /lychee-tmp folders exist ****
**** Create the symbolic link for the /uploads folder ****
**** Create the symbolic link for the /sym folder ****
**** Create the symbolic link for the /logs folder ****
**** Create the symbolic link for the /lychee-tmp folder ****
touch: cannot touch '/var/www/html/Lychee/storage/tmp/empty_file': No such file or directory
**** Copy the .env to /conf ****
**** Inject .env values ****
**** Generate the key (to make sure that cookies cannot be decrypted etc) ****

                                                                                
                           APPLICATION IN PRODUCTION.                           
                                                                                

   WARN  Command cancelled.  

**** Migrate the database ****

   INFO  Nothing to migrate.  

**** Make sure user.css exists and symlink it ****
**** Make sure custom.js exists and symlink it ****
**** Create user and use PUID/PGID ****
        User UID :      1000
        User GID :      1000
**** Make sure Laravel's log exists ****
**** Set Permissions ****
**** Start cron daemon ****
cron: unrecognized service

### Steps to reproduce the issue

system os： arm64
use docker:  
root@Chainedbox:~# docker inspect 0437a8c7f90e
[
    {
        "Id": "sha256:0437a8c7f90e4fad8c7b3094c59e85c7b4eed140b1110cecd63cdd41cdf332d7",
        "RepoTags": [
            "lycheeorg/lychee:dev"
        ],
        "RepoDigests": [
            "lycheeorg/lychee@sha256:38c64f130d96b68a57a0d2d7aebc284494a0235e33d867ca1391ae44e6a9c28b"
        ],
        "Parent": "",
        "Comment": "buildkit.dockerfile.v0",
        "Created": "2024-10-14T19:06:50.478578504Z",
        "Container": "",
        "ContainerConfig": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": null,
            "Cmd": null,
            "Image": "",
            "Volumes": null,
            "WorkingDir": "",
            "Entrypoint": null,
            "OnBuild": null,
            "Labels": null
        },
        "DockerVersion": "",
        "Author": "",
        "Config": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "ExposedPorts": {
                "80/tcp": {}
            },
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "PUID=1000",
                "PGID=1000",
                "USER=lychee",
                "PHP_TZ=UTC"
            ],
            "Cmd": [
                "nginx"
            ],
            "Healthcheck": {
                "Test": [
                    "CMD-SHELL",
                    "curl --fail http://localhost:80/ || exit 1"
                ]
            },
            "ArgsEscaped": true,
            "Image": "",
            "Volumes": {
                "/conf": {},
                "/logs": {},
                "/lychee-tmp": {},
                "/sym": {},
                "/uploads": {}
            },
            "WorkingDir": "/var/www/html/Lychee",
            "Entrypoint": [
                "/entrypoint.sh"
            ],
            "OnBuild": null,
            "Labels": {
                "maintainer": "lycheeorg"
            }
        },
        "Architecture": "arm64",
        "Os": "linux",
        "Size": 893722092,
        "VirtualSize": 893722092,
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/c611ed3598ac242858a57b8b0f47826b629f9e6c2502570cdd3771cce0c8cf77/diff:/var/lib/docker/overlay2/9c6b12acbd996182609ee46be2335bfdb60774b3a41623b014750026e992c959/diff:/var/lib/docker/overlay2/ddf701982f9064044c2b63db755c36eb1cc14e874173c49424ac198f2a0c95cb/diff:/var/lib/docker/overlay2/82c8e0f13b215cf934a34a6b285fb18a12b5a43679c86cf854fe28dbae2b588f/diff:/var/lib/docker/overlay2/cf7224c5f66955a3edee0aa281f75ddb5805ec158cfdf3e6b28afd5e5fe861f3/diff:/var/lib/docker/overlay2/429ac132b8e15a6e20a45a8029c51ec705f689f122d9900d07be52eb2b354fd2/diff",
                "MergedDir": "/var/lib/docker/overlay2/c935dde37f6340f82afd2ece5fbc16a2835af95736c8a7ad271460cb33644038/merged",
                "UpperDir": "/var/lib/docker/overlay2/c935dde37f6340f82afd2ece5fbc16a2835af95736c8a7ad271460cb33644038/diff",
                "WorkDir": "/var/lib/docker/overlay2/c935dde37f6340f82afd2ece5fbc16a2835af95736c8a7ad271460cb33644038/work"
            },
            "Name": "overlay2"
        },
        "RootFS": {
            "Type": "layers",
            "Layers": [
                "sha256:054df1200f3e1d70e2ebeafa03c965d0dd34e7dab90b4ea24963352e5590b573",
                "sha256:baaba4a1bd229c6e92978ba82083cb3af7502ab770c9997e99150744b7ad9c04",
                "sha256:a60d5a0c703d5c90892b38a11f6a096082c8556e1f70033604cc12d239aa2163",
                "sha256:2942c1b4be793bd5c8be74c9dff0a53cc54430fb46332bcff1624292d74f0863",
                "sha256:5f70bf18a086007016e948b04aed3b82103a36bea41755b6cddfaf10ace3c6ef",
                "sha256:c8b48bfb61aa5c02eb34e19e27feb1878a73718d9b99c4e3c4a0e7efc0b0c447",
                "sha256:5f70bf18a086007016e948b04aed3b82103a36bea41755b6cddfaf10ace3c6ef"
            ]
        },
        "Metadata": {
            "LastTagTime": "0001-01-01T00:00:00Z"
        }
    }
]

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by d7415** [comment]
The arm builds occasionally break like this and I have no idea why - it
doesn't seem to be repeatable. Hopefully the next rebuild will fix it. I'll
trigger a rebuild now. If that doesn't fix it I'll try looking deeper when
I have time.
